docs/additional-configs.md

```diff
@@ -332,8 +332,7 @@ the `feature-flags` ConfigMap alongside your Tekton Pipelines deployment via
 Features currently in "beta" are:
 
 | Feature                                                                                               | Proposal                                                                                                 | Alpha Release                                                        | Beta Release                                                         | Individual Flag               |
-|:------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------|:---------------------------------------------------------------------|:------------------------------|
-| [Array Results and Array Indexing](pipelineruns.md#specifying-parameters)                             | [TEP-0076](https://github.com/tektoncd/community/blob/main/teps/0076-array-result-types.md)              | [v0.38.0](https://github.com/tektoncd/pipeline/releases/tag/v0.38.0) | [v0.45.0](https://github.com/tektoncd/pipeline/releases/tag/v0.45.0) |                               |
```

Comment:
Looking from the preview, the format seems to be broken?

Reply:
Oh thanks! 👍